lib/components/form/call-taker/date-time-picker.tsx

```diff
@@ -134,7 +137,20 @@ const DateTimeOptions = ({
   // Update state when external state is updated
   useEffect(() => {
     if (initialDate !== date) setDate(initialDate)
-    if (initialTime !== time) setTime(initialTime)
+    if (initialTime !== time) {
+      setTime(initialTime)
+
+      // Don't update if still typing
+      if (timeRef.current !== document.activeElement) {
+        setTypedTime(
+          safeFormat(dateTime, timeFormat, {
```

Comment:
This needs to happen on initial render of the component (for situations such as pasting the URL into the browser), otherwise the "typed" time can be different from the status time as shown in the screenshot below.

<img width="173" alt="image" src="https://github.com/opentripplanner/otp-react-redux/assets/56846598/9c593617-dd3e-4b7a-a785-ffa88177335b">

Reply:
Good catch thanks! Let me add an initial load update

Reply:
Try 3ccd8d23